specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2019-12-01-preview/containerregistry.json

```diff
@@ -0,0 +1,1291 @@
+{
```

Comment:
There are a number of nested resource types from the prior version not carried forward. Is that intentional? It's a suboptimal experience for the customer to produce a template to configure a resource if they have to use different API versions to configure different element/sub-resources.

Reply:
Added the deleted controllers back due to consistency.